fl4health/privacy_mechanisms/dpBinomial.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+import torch
+
+
+# TODO why doesn't mechanism result in output type List[int]
+def BinomialMechanism(query_vector: List[int], N: int, p: float, j: int) -> List[int]:
```

Comment:
We've generally tried to adhere to the PEP 8 guidelines for casing of things like classes and functions (though we're definitely not perfect at it). If you're curious about the standard you can take a look [here](https://peps.python.org/pep-0008/) under sections like "[Class Names](https://peps.python.org/pep-0008/#class-names)". 

The two most common to know are 
- functions are lowercase with "words" separated by underscores (i.e. `binomial_mechanism`)
- class names are "camel cased" (i.e. `FedProxStrategy`)

Reply:
Note that the casing generally overrides something being a proper noun (i.e. `gaussian_mechanism` over `Gaussian_mechanism`)